roles/atomic_upgrade/tasks/main.yml

```diff
@@ -35,9 +35,8 @@
 - block:
 
     - name: Workaround BZ1290281
-      selinux:
-        policy: targeted
-        state: permissive
+      # Only temporary, don't touch /etc/sysconfig/selinux
```

Comment:
it's the right bug, well one of them at least.  I don't understand how os-tree / atomic works well enough to explain/comment about the bug.  All I know is that's the one in play here (for older atomic hosts).  If you understand better or can suggest a comment, I'll happily add one.